### DISCUSSION TITLE
Is there a way to prevent eslint-mdx to parse files with other extensions?

### DISCUSSION THREAD

**Discussion by silversonicaxel** [mdx-js/mdx#1308] [Q&A] [answered]
Hello, I have a react project, with eslint, we started to use mdx files and I wanted to lint them also.
I found https://github.com/mdx-js/eslint-mdx library and I thought it could fit in my project.
So I installed it adding the dependency in package.json
```
    "eslint-plugin-mdx": "1.8.2",
```

and just extending the eslint configuration
```
  extends: [
    ....,
    'plugin:mdx/recommended',
]
```

At this moment, I run my current eslint command (COMMAND1)
```
eslint --fix --ext .js,.jsx,.json,.ts,.tsx . 
```

ready to update it to this one (COMMAND2)
```
eslint --fix --ext .js,.jsx,.json,.ts,.tsx,.mdx . 
```

But I've some new issues appeared already running the eslint without touching the `.mdx` files (COMMAND1)

```
/MY_PRIVATE_PATH/MY_PRIVATE_FILE.js
  28:3  error  Expected an assignment or function call and instead saw an expression  mdx/no-unused-expressions
```

and more errors occur when I execute it including the `.mdx` files (COMMAND2)

I'm wondering why this happens, should `plugin:mdx/recommended` work towards `.mdx` files?

**Comment by ChristianMurphy** [comment, marked as answer]
@silversonicaxel make sure you have the setup correct for the version on ESLint you are using https://github.com/mdx-js/eslint-mdx#usage
If the issue persists see https://github.com/mdx-js/eslint-mdx/issues/167#issuecomment-596133414

**Reply by silversonicaxel** [reply]
my eslint version is `"eslint": "^7.10.0",` and I followed the suggestions for `eslint >= 6.4.0` but actually it worked the tip for `eslint >=6.0.0 and <6.4.0` and now it works super fine. Weird but thanks @ChristianMurphy
